src/main/java/reposense/commits/CommitResultAggregator.java

```diff
@@ -39,60 +42,74 @@ public static CommitContributionSummary aggregateCommitResults(
     }
 
     private static Map<Author, Float> calcAuthorContributionVariance(
-            Map<Author, List<AuthorDailyContribution>> intervalContributionMaps) {
+            Map<Author, List<AuthorDailyContribution>> intervalContributionMaps, Date startDate) {
         Map<Author, Float> result = new HashMap<>();
         for (Author author : intervalContributionMaps.keySet()) {
             List<AuthorDailyContribution> contributions = intervalContributionMaps.get(author);
-            result.put(author, getContributionVariance(contributions));
+            result.put(author, getContributionVariance(contributions, startDate));
         }
         return result;
     }
 
-    private static float getContributionVariance(List<AuthorDailyContribution> contributions) {
+    private static float getContributionVariance(List<AuthorDailyContribution> contributions, Date startDate) {
         if (contributions.size() == 0) {
             return 0;
         }
         //get mean
         float total = 0;
+        long totalDays = ((lastDate.getTime()
+                - getStartOfDate(startDate).getTime()) / DAYS_IN_MS) + 1;
```

Comment:
Since division has priority over addition, can remove 1 parenthesis to make it more readable.
E.g. 
```
long totalDays = (lastDate.getTime() - getStartOfDate(startDate).getTime()) 
        / DAYS_IN_MS + 1;
```